frontend/src/scenes/feature-flags/FeatureFlag.tsx

```diff
@@ -504,6 +504,12 @@ function UsageTab({ featureFlagKey }: { id: string; featureFlagKey: string }): J
             value: featureFlagKey,
             operator: PropertyOperator.Exact,
         },
+        {
+            key: '$feature_flag_response',
```

Comment:
seems like a bug, do you know when this is empty? 🤔

Reply:
Not sure what you mean

Reply:
> there are a lot of feature flag called events that aren't accompanied by a property

meaning when feature_flag_response is empty? - do you know why?

Reply:
(not a bug in the PR lol, this is a go, I meant in general with sdks)

Reply:
oh, it might be for us in particular but we're loading flags differently because we don't use posthog.getFeatureFlag. We have feature flag called events that don't have a response (either in $feature_flag_response or $feature/<key>). This patch won't help us. it'll just reduce noise